translations/en.json

```diff
@@ -510,7 +510,9 @@
 				"no-results": "No shorts found",
 				"accrued-interest": "Accrued Interest"
 			}
-		}
+		},
+		"l1-deprecated": "Shorting on L1 is no longer available; please switch to L2, or manage your L1 positions below."
```

Comment:
Can we change "is no longer available" to "is deprecated"?

Reply:
done